src/asset-swapper/constants.ts

```diff
@@ -58,14 +58,8 @@ const DEFAULT_SWAP_QUOTER_OPTS: SwapQuoterOpts = {
 const DEFAULT_EXCHANGE_PROXY_EXTENSION_CONTRACT_OPTS: ExchangeProxyContractOpts = {
     isFromETH: false,
     isToETH: false,
-    affiliateFees: [
-        {
-            feeType: AffiliateFeeType.None,
-            recipient: NULL_ADDRESS,
-            buyTokenFeeAmount: ZERO_AMOUNT,
-            sellTokenFeeAmount: ZERO_AMOUNT,
-        },
-    ],
+    sellTokenAffiliateFees: [],
+    buyTokenAffiliateFees: [],
```

Comment:
Nice! With this we don't even need a feature flag to control which affiliate fee to charge